.gitignore

```diff
@@ -43,3 +43,4 @@ com.ibm.jvmti.tests
 # Default test directory (BUILD_ROOT)
 /jvmtest/
 /build/
+.vscode/
```

Comment:
Is this directory normally at the root of the working copy? If so, it should be `/.vscode/` here. Please also add a comment so it's clear it's not related to tests.

Reply:
updated... the tool automatically added it without the `/` which I would have preferred to keep but convention in this file says to add it so I did